momentum_scanner_intraday.py

```diff
@@ -77,22 +115,31 @@ def find_liquidity(coin, dex):
 def find_best_liquidity(coin, dex):
     best_volume = 0
     best_pair = ""
-    for ticker in gecko.query_coin(coin)["tickers"]:
+    best_reserve = 0
+    re = gecko.query_coin(coin)
+    for ticker in re["tickers"]:
         if ticker["market"]["identifier"] == dex:
             pair = ticker["target_coin_id"] + "<>" + ticker["coin_id"]
             volume = float(ticker["converted_volume"]["usd"])
             if volume > best_volume:
                 best_pair = pair
                 best_volume = volume
 
-    return best_volume, best_pair
+    chain_cg, chain_cgterminal = find_chain(dex)
+    if chain_cg in re['platforms'].keys():
```

Comment:
```suggestion
    if chain_cg in re['platforms']:
```

Reply:
Why do you suggest that? re['platforms'] is a dictionary

Reply:
```
In [1]: 'a' in {'a': 1}
Out[1]: True
```
This is how you test for keys existing in a dictionary in python.

Reply:
Changed as suggested. Learned something new